frontend/rolecall/src/app/login/welcome-page.component.html

```diff
@@ -15,8 +15,7 @@ <h1 style="color:#FFFFFF; font-size:800%; font-weight:200; font-family:Tahoma, G
           <!-- Divider -->
           <hr>
           <!-- Description -->
-          <!-- TODO: Fix invalid font-size pixel value. -->
-          <h2 style="color:#FFFFFF; font-size: 200 px; font-weight: 200; font-family:Tahoma, Geneva, sans-serif">
+          <h2 style="color:#FFFFFF; font-size: 200%; font-weight: 200; font-family:Tahoma, Geneva, sans-serif">
```

Comment:
Please double check that 200% was intended here instead of 200px.

Reply:
It was intended.